documentation/src/main/wiki/Troubleshooting.md

```diff
@@ -0,0 +1,51 @@
+# Troubleshooting
+This section contains tips and ways to repair AET instances. If all the tips from the list failed
+or you have discovered a new one, please raise a ticket using [Issues Tool](https://github.com/Cognifide/aet/issues)
+or create a Pull Request with change to [this file](https://github.com/Cognifide/aet/blob/master/documentation/src/main/wiki/Troubleshooting.md).
+
+- [Karaf](#karaf)
+  - [Karaf can't find some dependencies or configurations - clearing the cache](#karaf-cant-find-some-dependencies-or-configurations---clearing-the-cache)
+- [ActiveMQ](#activemq)
+  - [Messages serialization issue](#messages-serialization-issue)
+- [MongoDB](#mongodb)
+  - [Make sure that you have indexed `metadata` collection in the project db.](#make-sure-that-you-have-indexed-metadata-collection-in-the-project-db)
+- [Report app](#report-app)
+  - [Failed to load report data](#failed-to-load-report-data)
+
+## Karaf
+
+### Karaf can't find some dependencies or configurations - clearing the cache
+This may help in multiple situations, e.g. new version of AET was deployed
+or Karaf can't find some dependencies or configurations.
+To clear bundles cache, stop Karaf, remove following directories from `$KARAF_HOME/data` and start Karaf:
+- `cache`
+- `generated-bundles`
+- `tmp`
```

Comment:
We can delete all content from *data* folder (with copying logs folder to backup location if needed).
I would suggest following set of commands:

```
sudo service karaf stop
sudo rm -fr /opt/aet/karaf/current/data/*
sudo service karaf start
```